js/app.js

```diff
@@ -0,0 +1,43 @@
+const totalZombies = document.querySelector('.total-zombies'),
+      deadZombies = document.querySelector('.dead-zombies'),
+      smallZombies = document.querySelector('.small-zombies'),
+      madZombies = document.querySelector('.mad-zombies'),
+      strongZombies = document.querySelector('.strong-zombies');
```

Comment:
please define a keyword for each variable
```
const totalZombies;
const deadZombies;
...
```

Reply:
fixed